### PR TITLE
CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Brijeshthummar02**
issue #7

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Code of Conduct  
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers of **WildGuard** pledge to make participation in our project and community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, education, socioeconomic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (mainak55512) [inline CODE_OF_CONDUCT.md]: Please change the name from 'wildguard' to 'rjq'

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+We are committed to creating a friendly and inclusive space where everyone can contribute and collaborate on animal welfare and environmental conservation efforts.
```
Comment (mainak55512) [inline CODE_OF_CONDUCT.md]: Wrong sentence, please modify it according to the project

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- Acknowledging the contributions of others graciously.
+- Supporting others and celebrating successes as a community.
+
+Examples of unacceptable behavior include:
+
+- The use of sexualized language or imagery and unwelcome sexual attention or advances.
+- Trolling, insulting, derogatory comments, and personal or political attacks.
+- Public or private harassment of any kind.
+- Publishing others’ private information without explicit permission.
+- Any behavior that is not aligned with the mission of **WildGuard**.
```
Comment (mainak55512) [inline CODE_OF_CONDUCT.md]: WildGuard -> rjq

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable behavior and will take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, issues, or other contributions that do not align with this Code of Conduct, and they may temporarily or permanently ban any contributor whose behavior they deem inappropriate.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing **WildGuard** or its community. Examples of representation include using an official project email address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
```
Comment (mainak55512) [inline CODE_OF_CONDUCT.md]: WildGuard -> rjq

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at **[your-email@example.com]**. All complaints will be reviewed and investigated, and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident.
```
Comment (mainak55512) [inline CODE_OF_CONDUCT.md]: Change your-email@example.com -> mainak.bhattacharjee432@gmail.com

**Review by mainak55512 — CHANGES_REQUESTED** [review]
Hi 👋,

I would like to have some changes to be made in the COC. Could you please make the changes.

Thanks in advance 😃

**Comment by mainak55512** [comment]
@Brijeshthummar02 also please accomodate COC in Readme file

**Comment by Brijeshthummar02** [comment]
@mainak55512 previous file was by mistake written wrong, i've replaced whole file with new lines and changes, please go through it.

**Comment by Brijeshthummar02** [comment]
good to go??

**Comment by mainak55512** [comment]
> good to go??

Merged 👍

**Pull request merged**